Dockerfile

```diff
@@ -14,7 +14,7 @@ FROM base as production
 RUN poetry install --no-dev
 COPY todo_app todo_app
 
-ENTRYPOINT poetry run gunicorn todo_app.wsgi:start -b 0.0.0.0:80
+CMD poetry run gunicorn "todo_app.wsgi:start" -b 0.0.0.0:$PORT
```

Comment:
Note that this probably breaks your Readme instructions since they don't expect the PORT to be required to be passed through; a nice solution can be to add an `ENV PORT=80` line just above this that sets a default value to 80 as before (so preserving backwards compatibility) but it will be overridden if the user provides a different value at run time